blottertrax/helper/repost_checker.py

```diff
@@ -0,0 +1,57 @@
+import sqlite3
+
+
+class RepostChecker:
+    cursor = None
+    sql = None
+
+    @classmethod
+    def __init__(self):
+        self.sql = sqlite3.connect('{}/../{}/../database/repost.db')

```

Comment:
Shouldn't need the {} braces if we aren't formatting anything.

Reply:
You would think, but in reality I think I did try that and I think it didn't work. I'll give it another test tomorrow